config/locales/errors/fr.yml

```diff
@@ -102,6 +102,9 @@ fr:
       pwned_password: Le mot de passe que vous avez entré n’est pas sécurisé. C’est
         dans une liste de mots de passe connus exposés dans les violations de
         données.
+      too_short:
+        one: est trop courte (le minimum est de 1 caractère)
+        other: est trop courte (le minimum est de %{count} caractères)
```

Comment:
I think this is probably a good change to make if it's a standard Rails error message, but based on the screenshots where this is shown, it seems a little strange we're just showing a fragment of text "is too short" under the field. It's also an issue in that the adjective translates differently depending on the gender of the noun (e.g. in French, [`court` vs. `courte`](https://translate.google.com/?sl=en&tl=fr&text=the%20username%20is%20too%20short.%0Athe%20question%20is%20too%20short.&op=translate)). Separately, maybe we could seek to improve this text to be more specific, like "The password is too short".

Reply:
I like this idea, may want to talk to someone from content to look into this. Good catch Andrew!

Reply:
This is a good idea, but changing the translation here means that all "too short" validations will say "Password is too short"

Reply:
> This is a good idea, but changing the translation here means that all "too short" validations will say "Password is too short"

Oh yes, sorry, I should have been more clear that I was meaning to create a separate string for us to use for the specific case where passwords are too short. As @mitchellhenke points out, the changes in cee4ade will cause this text to be used everywhere a field value is too short, not just passwords.

I'd also be fine to split this out as a separate task if it makes more sense to do so.